packages/firestore/src/core/target.ts

```diff
@@ -22,11 +22,14 @@ import {
   Bound,
   boundEquals,
   canonifyBound,
-  canonifyOrderBy,
-  Filter,
+  canonifyFilter,
+  filterEquals,
+  stringifyFilter,
   OrderBy,
   orderByEquals,
-  stringifyOrderBy
+  stringifyOrderBy,
```

Comment:
I think we've discussed this before, so I won't push, but I find the difference in ordering (subject-operation vs operation-subject) between `orderByEquals` and `stringifyOrderBy` to be jarring. Autocompletion and muscle memory would be much easier if there was just one way to do it.

If these were all in the form subject-operation they would group nicely in imports as well.

What do you think of these?
```
orderByEquals
orderByToString
orderByCanonicalId
```

Reply:
The naming in this PR is inspired by the names in the Values helper. I am also not a big fan how "equals" is an outlier, but _on its own_ I do think that each name  looks ok. For that reason, I am kind of reluctant to change it right now, but I am not opposed to overhauling all stringify/canonicalize methods in a follow up.